controller/Synchronizer.php

```diff
@@ -123,6 +130,24 @@ protected function getLastSyncTask()
         return null;
     }
 
+    /**
+     * @return int
+     */
+    protected function getActiveSessions()
+    {
+        /** @var DeliveryMonitoringService $deliveryMonitoringService */
+        $deliveryMonitoringService = $this->getServiceLocator()->get(DeliveryMonitoringService::SERVICE_ID);
+        $deliveryExecutionsData = $deliveryMonitoringService->find([
+            DeliveryMonitoringService::STATUS => [
+                DeliveryExecution::STATE_ACTIVE
+            ]
+        ]);
+        if ($deliveryExecutionsData) {
+            return sizeof($deliveryExecutionsData);
```

Comment:
Please use `count` instead of `sizeof` in the newest php version 7.2 count it's much faster.

Reply:
anyway, you can use the `count` from interface and pass the criteria
like https://github.com/oat-sa/extension-tao-proctoring/blob/33f52a605b716101bf07bcc109902943c77e00c1/model/monitorCache/implementation/MonitoringStorage.php#L323